src/frida.vala

```diff
@@ -1143,6 +1143,12 @@ namespace Frida {
 			}
 		}
 
+		private class QuerySystemParametersTask : DeviceTask<HashTable<string, Variant>?> {
+			protected override async HashTable<string, Variant>? perform_operation () throws Error, IOError {
+				return yield parent.query_system_parameters (cancellable);
+			}
+		}
```

Comment:
This should be moved to the two methods it belongs to, following the order we're using for other methods.